### PR TITLE
Branch user guide

### PR DESCRIPTION
**Pull request by Cp-John**
Add sorting and removing parts in the user guide.

### CODE REVIEW

**Review by w2vgd — COMMENTED** [review]
LGTM 👍 Can you update the command summary at the bottom of the user guide page for the 2 commands before we merge this?

**Comment by Cp-John** [comment]
Sure!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -213,4 +246,6 @@ Action | Format, Examples
 **Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
 **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
 **List** | `list`
+**Sort** | `sort`<br> e.g., `sort appointment asc deadline`
```
Comment (w2vgd) [inline docs/UserGuide.md]: Should there be 2 formats for the sort command as well?

Comment (Cp-John) [reply docs/UserGuide.md]: Updated already

**Review by w2vgd — APPROVED** [review]
LGTM 👍

**Pull request merged**